flocx_market/models/offer.py

```diff
@@ -0,0 +1,53 @@
+from db import db
+from db import sqlalchemy_jsonfield
```

Comment:
This should just be:

```
import sqlalchemy_jsonfield
```

And you need to add `sqlalchemy_jsonfield` to `requirements.txt`.

Reply:
sounds good, 
probably this spelling didn't work well "SQLAlchemy-JSONField>=0.8.0"

Reply:
That spelling *should* work, actually.  So maybe just fix the `import`.

Reply:
ok, I'll look into it